### PR TITLE
Improve histograms performance

### PR DESCRIPTION
**Pull request by GabiThume**
Use `d3-color` package to improve performance when calculating histograms.

### CODE REVIEW

**Comment by GabiThume** [comment]
@automata review, please?

**Comment by jonnor** [comment]
@GabiThume what is the performance like, before and after?

**Comment by GabiThume** [comment]
@jonnor For a 1024x768 image it was around 25s before and less than 5s after.

File: components/GetHistogram.coffee [hunk 1]
```diff
@@ -92,7 +91,7 @@ exports.getComponent = ->
       y: zero new Array 256
       h: zero new Array 361 # degrees [0,0, 360.0] -> [0, 361]
       s: zero new Array 101 # [0.0, 1.0] -> [0, 101]
-      c: zero new Array 101 # ?
+      c: zero new Array 135 # ?
```
Comment (jonnor) [inline components/GetHistogram.coffee]: Why this change?

Comment (GabiThume) [inline components/GetHistogram.coffee]: Because it was not instantiated the right amount of dimensions before (https://github.com/gka/chroma.js/issues/63).
